src/rpc/blockchain.cpp

```diff
@@ -1924,6 +1924,28 @@ static UniValue savemempool(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+static UniValue clearmempool(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0) {
+        throw std::runtime_error(
+            "clearmempool\n"
+            "\nRemoves all transactions from the mempool.\n"
+            "It will fail if a previous dump is still being loaded.\n"
+            "\nExamples:\n"
+            + HelpExampleCli("clearmempool", "")
+            + HelpExampleRpc("clearmempool", "")
+        );
+    }
+
+    if (!g_is_mempool_loaded) {
+        throw JSONRPCError(RPC_MISC_ERROR, "The mempool was not loaded yet");
+    }
+
+    mempool.clear();
+
+    return NullUniValue;
```

Comment:
Could return the number of evicted txns to be more useful.

Reply:
Then return an object which can have more keys in the future.

Reply:
Yes, that makes sense.  If you generally think having this method is useful, I'll add a useful return value to it (e.g. starting with the number of evicated tx).

Reply:
Changed the RPC to return a JSON object, with `nTx` holding the number of evicted transactions.